airdrop/job/reward_processors/nunet_reward_processor.py

```diff
@@ -11,15 +10,17 @@
 
 TOKENS_ALLOCATED_PER_WINDOW = Decimal(12500000)
 SCORE_DENOMINATOR = Decimal(100000)
+NUNET_DECIMALS = Decimal(1000000)
```

Comment:
We should keep all these values as part of constant.py

Reply:
True but these are specific to the handler which is dynamic. Tomorrow I will have a new handler for the next airdrop so want to keep all dependencies localized to one file rather than add a constants.py for each handler.